pkg/client/cluster-essentials/pkg/utils/utils.go

```diff
@@ -0,0 +1,265 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package utils contain cluster essential util APIs
+package utils
```

Comment:
Looking at the usage of functions in this package, all of it is in `clusterinit.go`. Why not just move this file to that package and get rid of `utils` package?

Reply:
Please see https://github.com/vmware-tanzu/tanzu-framework/pull/4503#discussion_r1157914392 for desired package structure.